terraform/workspace-variables/production.tfvars

```diff
@@ -0,0 +1,15 @@
+# Platform
+environment = "production"
+app_environment = "production"
+
+# Gov.UK PaaS
+paas_api_url = "https://api.london.cloud.service.gov.uk"
+paas_space_name = "early-careers-framework-prod"
+paas_postgres_service_plan = "small-11"
+paas_app_start_timeout = "180"
+paas_app_stopped = false
+paas_web_app_deployment_strategy = "blue-green-v2"
+paas_web_app_instances = 1
+paas_web_app_memory = 8192
+paas_worker_app_instances = 1
+paas_worker_app_start_command = "bundle exec rake jobs:work"
```

Comment:
I assume this is deliberately different?

Reply:
Yeah, I haven't played with mailer pools yet, will change it when I'm sure we want it :)

Reply:
Just for future reference, if you ever call `deliver_later` that should end up in the mailers pool. It will still get send this way, I just didn't want it being blocked by a long running task on ours